lib/DataStore.js

```diff
@@ -51,6 +53,18 @@ function DataStore(userStore, roomStore) {
         unique: true,
         sparse: false
     }, errLog("user id"));
+
+    this._privateKey = null;
+
+    if (pkeyPath) {
+        try {
+            this._privateKey = fs.readFileSync(pkeyPath, "utf8").toString();
+            log.info(`Private key loaded from ${pkeyPath} - IRC password encryption enabled.`);
+        }
+        catch (err) {
+            log.error('Could not load private key ${err.message}.');
```

Comment:
But what if someone wants to use the bridge without bothering with generating a PEM key?